be-traveltrove/src/main/java/com/traveltrove/betraveltrove/utils/DatabaseLoader.java

```diff
@@ -17,21 +22,22 @@
 @Slf4j
 public class DatabaseLoader {
 
+    // Countries :)
     private final CountryRepository countryRepository;
 
     @PostConstruct
     public void loadData() {
         List<Country> sampleCountries = List.of(
-                new Country(null, generateUUIDString(), "United States", "usa.png"),
-                new Country(null, generateUUIDString(), "Canada", "canada.png"),
-                new Country(null, generateUUIDString(), "France", "france.png"),
-                new Country(null, generateUUIDString(), "Germany", "germany.png"),
-                new Country(null, generateUUIDString(), "Italy", "italy.png"),
-                new Country(null, generateUUIDString(), "Japan", "japan.png"),
-                new Country(null, generateUUIDString(), "Brazil", "brazil.png"),
-                new Country(null, generateUUIDString(), "Australia", "australia.png"),
-                new Country(null, generateUUIDString(), "India", "india.png"),
-                new Country(null, generateUUIDString(), "China", "china.png")
+                new Country(null, "ad633b50-83d4-41f3-866a-26452bdd6f33", "United States", "usa.png"),
+                new Country(null, "b1db23af-4f2d-4138-b623-e906f5287e90", "Canada", "canada.png"),
+                new Country(null, "118495e2-e1ce-4987-bb09-a95f93515bda", "France", "france.png"),
+                new Country(null, "cf09374e-6d51-47ec-91e1-2b69a788457f", "Germany", "germany.png"),
+                new Country(null, "dde33653-fc58-457c-9d33-a322a2a82835", "Italy", "italy.png"),
+                new Country(null, "f05f8f2c-f86d-4cf5-8a9b-19e70f1b96d5", "Japan", "japan.png"),
+                new Country(null, "b511c7ee-c0f3-4902-9bfa-d50f59ef2b3c", "Brazil", "brazil.png"),
+                new Country(null, "d5d4e5e2-f960-4cf4-beed-c38104e5d5b4", "Australia", "australia.png"),
+                new Country(null, "3cd2ad86-26cc-42ad-8b20-8b0b6e6d2a2e", "India", "india.png"),
+                new Country(null, "877ec1c0-ffab-449e-a2ec-08f95db58f55", "China", "china.png")
```

Comment:
Interesting implementation of the hard coded generatedUUIDString, it allows the codes to be permanent regardless of the run, which makes it easier for testing.